Makefile

```diff
@@ -4,6 +4,8 @@ CGO_ENABLED=0
 GOOS=linux
 CORE_IMAGES=./cmd/activator ./cmd/autoscaler ./cmd/autoscaler-hpa ./cmd/controller ./cmd/queue ./cmd/webhook ./cmd/networking/nscert ./vendor/knative.dev/pkg/apiextensions/storageversion/cmd/migrate
 TEST_IMAGES=$(shell find ./test/test_images -mindepth 1 -maxdepth 1 -type d)
```

Comment:
Can't you "just" adjust the maxdepth to 2 here? Worked for me locally at least on a first glance.

Reply:
Unfortunately it includes some unnecessary directory :crying_cat_face: 

```
$ find ./test/test_images -mindepth 1 -maxdepth 2 -type d
./test/test_images/pizzaplanetv1
./test/test_images/grpc-ping
./test/test_images/grpc-ping/proto                   <- *
./test/test_images/autoscale
./test/test_images/observed-concurrency
./test/test_images/helloworld
./test/test_images/multicontainer                    <- *
./test/test_images/multicontainer/sidecarcontainer
./test/test_images/multicontainer/servingcontainer
./test/test_images/runtime/handlers/      
 ...
```
We can ignore error on `go install` and `ko resolve` but perhaps it is better to use accurate directories rather than ignoring?

Reply:
Ahhh, I see. I figured you'd tried this. Any reason we're not replacing the entire `TEST_IMAGES` variable with the new ones? If so, please document the reason in a comment.

Reply:
As chat on Slack, maxdepth 2 does not produce error but it creates extra `Dockerfile`. So updated to use `find ./test/test_images ./test/test_images/multicontainer -mindepth 1 -maxdepth 1 -type d`. Thank you.